File: java/app/src/main/java/algo/exercises/preparation/BabyBites.java
package algo.exercises.preparation;

import java.util.Scanner;

public class BabyBites {
    
    public static void main(String[] args) {
        Scanner scanner = new Scanner(System.in);
        int n = scanner.nextInt();
        scanner.nextLine();
        String[] words = scanner.nextLine().split(" ");
        scanner.close();

        String res = "makes sense";
        for (int i = 0; i < n; i++) {
            if (!words[i].equals("mumble") && Integer.parseInt(words[i]) != i + 1) {
                res = "something is fishy";
                break;
            }
        }
        System.out.println(res);
    }
}

File: python/exercises/preparation/BabyBites.py
from typing import List

def baby_bites() -> None:
    n: int = int(input())
    words: List[str] = input().split(' ')
    res = True
    for i in range(n):
        if not words[i] == "mumble" and int(words[i]) != i + 1:
            res = False
    print("makes sense" if res else "something is fishy")

baby_bites()
    